tests/framework/extensions/prime/primechecks.go

```diff
@@ -0,0 +1,92 @@
+package primechecks
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	client "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/extensions/rancherversion"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	PodResourceSteveType = "pod"
+	rancherImage         = "rancher"
+)
+
+// CheckUIBrand checks the UI brand of Rancher Prime. If the Rancher instance is not Rancher Prime, the UI brand should be blank.
+func CheckUIBrand(client *rancher.Client, isPrime bool, brand *client.Setting, rancherBrand string) (string, error) {
+	if isPrime {
+		if rancherBrand == brand.Value {
```

Comment:
You can shortened this logic by doing `if isPrime && rancherBrand == brand.Value` I don't get why you have an error return if it never returns an error. Why not return an error if this check fails?

Reply:
I can remove the errors in the return since these are just returning strings, that was an oversight.

Reply:
Removed the error as a return value and just kept it as a string being removed.